cql3/statements/modification_statement.cc

```diff
@@ -241,6 +244,12 @@ static thread_local inheriting_concrete_execution_stage<
 
 future<::shared_ptr<cql_transport::messages::result_message>>
 modification_statement::execute(query_processor& qp, service::query_state& qs, const query_options& options) const {
+    return execute_without_checking_exception_message(qp, qs, options)
+            .then(cql_transport::messages::propagate_exception_as_future<shared_ptr<cql_transport::messages::result_message>>);
+}
```

Comment:
Is there an explanation why this did not add an allocation+task? The future returned by execute_without_checking_exception_message() could not be ready, and so this adds a continuation.

Perhaps we eliminated a continuation from some other place.

Reply:
Oh wait, the regular path calls execute_without_checking_exception_message().